ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -145,6 +145,10 @@ http://www.freedesktop.org/software/systemd/man/systemd.exec.html
 valid for [Service], [Socket], [Mount], or [Swap] sections
 }
 type ${project.artifactId}_unitfile_config_systemd_exec = {
+    'CacheDirectoryMode' ? type_octal_mode
+    'CacheDirectory' ? string[]
```

Comment:
Looks like systemd only accepts relative paths here, and `..` is banned - should we have a type that enforces that?

Reply:
I can add something like:

```
# type for a relative directory (no leading /) and may not include ".."
type ${project.artifactId}_relative_directory = string with !match(SELF, '(^/|\.\.)');
```

But currently we don't really check anything like this. The `WorkingDirectory` and `RootDirectory` are also just strings.

Reply:
Yep,  I think that would make a lot of sense if you can add it easily enough.

Reply:
Done. Please have a look.